enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -109,7 +108,16 @@ async function metabaseAuthHandler(req: Request, res: Response) {
 
 const app = express()
 
-// middleware
+// Middleware
+
+// If your FE application is on a different domain from your BE, you need to enable CORS
+// by setting Access-Control-Allow-Credentials to true and Access-Control-Allow-Origin
+// to your FE application URL.
+app.use(cors({
+  origin: FRONTEND_URL,
```

Comment:
This wasn't necessary for me, but they were both running on localhost with different ports. Is this needed?

Reply:
Tested. You're right it's [`true` by default](https://expressjs.com/en/resources/middleware/cors.html#configuring-cors-w-dynamic-origin) (See "Configuration options") which means it will set to whatever the origin of the request.

Reply:
Fixed in 1bed7c466335bd7456d6a3c0bbeb589cc28f1029